src/components/Navbar/index.jsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
-import {Navbar, NavbarBrand, NavbarContent, NavbarItem, Link, Button} from "@nextui-org/react";
+import { Navbar, NavbarBrand, NavbarContent, NavbarItem, Link, Button } from "@nextui-org/react";
 import { AcmeLogo } from "../../assets/AcmeLogo";
```

Comment:
Is this AcmeLogo used anywhere? If not, you can delete it :)

Reply:
It's used in the Navbar it's the "logo". We thought it would be best to delete it when we have the new one :)